### ISSUE TITLE
Breaking with Google PubSub >= 2.0

### ISSUE THREAD

**Issue by andrewgy8** [mercadona/rele#184]
In the 1.0 release, I had to pin pubsub to <2.0. Otherwise, our tests would break. It seems there were major changes, and we need to update some usages in our code. 

To reproduce: 
1. update google pubsub library to >2.0. 
1. Run tests

**Comment by mohit2494** [comment]
Hi there,
I've come here from hacktoberfest. Can someone please explain what needs to be done. 
Best
Mohit

**Comment by andrewgy8** [comment]
Sure @mohit2494 !

This is to make the lib compatible with the latest release of `google-cloud-pubsub` which introduced breaking changes that we are unable to handle right now. 

[Here](https://github.com/mercadona/rele/blob/master/setup.py#L42) you will see that we pin `google-cloud-pubsub<2.0`. That should be removed or at the very least changed to <2.1 (according to the conversation in #185 .

Basically to solve this, you will need to read the docs of `google-cloud-pubsub` and make the necessary changes to the code/tests so that we can install Relé with >2.0.

**Comment by luisgc93** [comment]
There seems to be a new way of retrieving the topic path in v2.0.

Before we could do: 
````
    subscriber = pubsub_v1.SubscriberClient()
    topic_path = subscriber.topic_path(project_id, topic_id)
````
Whereas now the SubscriberClient doesn't have this attribute anymore so the following is done instead:

`````
    publisher = pubsub_v1.PublisherClient()
    subscriber = pubsub_v1.SubscriberClient()	    subscriber = pubsub_v1.SubscriberClient()
    topic_path = subscriber.topic_path(project_id, topic_id)	    topic_path = publisher.topic_path(project_id, topic_id)
``````
